<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM BAGIAN</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="kodeBagianLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kode Bagian</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="kodeBagianLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaBagianLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nama Bagian</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="namaBagianLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="gajiPokokLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gaji Pokok</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="gajiPokokLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="uangTransportLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uang Transport</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="uangTransportLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="uangMakanLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uang Makan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="uangMakanLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="uangLemburLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uang Lembur</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="uangLemburLineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableBagian">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>641</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
